code/game/machinery/cryo.dm

```diff
@@ -76,19 +76,33 @@
 
 /obj/machinery/atmospherics/unary/cryo_cell/can_crawl_through()
 	return 0 // No. Just no.
-	
+
 /obj/machinery/atmospherics/unary/cryo_cell/allow_drop()
 	return 0
 
 /obj/machinery/atmospherics/unary/cryo_cell/relaymove(var/mob/user)
 	open_machine()
 
-/obj/machinery/atmospherics/unary/cryo_cell/container_resist()
-	if(stat & DEAD)
-		return
-	sleep(usr.stat * 1200)
-	open_machine()
-	return
+/obj/machinery/atmospherics/unary/cryo_cell/verb/move_eject()
+	set name = "Eject Cryo Cell"
+	set desc = "Begin the release sequence inside the cryo tube."
+	set category = "Object"
+	set src in oview(1)
+	if(usr == occupant || contents.Find(usr))	//If the user is inside the tube...
```

Comment:
Redundant contents.Find. Only one person can be in a cryogenics tube at a time, and if they're not the occupant then they're not in the tube.